magpie/utils.py

```diff
@@ -26,8 +25,7 @@
         Any, AnyKey, Str, List, Optional, Type, Union,
         AnyResponseType, AnyHeadersType, LoggerType, CookiesType, SettingsType, AnySettingsContainer,
     )
-    # noinspection PyProtectedMember
-    from typing import _TC  # noqa: F401
+    from typing import _TC  # noqa: F401,W0212
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Imports from package typing are not grouped](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)